src/anm/analytics.js

```diff
@@ -16,27 +13,15 @@ var Analytics = function () {
             var data = JSON.stringify(self.queue);
             self.queue = [];
 
-            if (supportSendBeacon) {
+            if (navigator.sendBeacon) {
                 navigator.sendBeacon(animatronUrl, data);
             } else {
-                sendViaGif(data);
+                engine.ajax(animatronUrl, null, null, 'POST', null, data, false);
             }
         }
-        if (!supportSendBeacon) {
-            setTimeout(event, timeout);
-        }
     };
 
-    var sendViaGif = function (data) {
-        if (!beacon) {
-            beacon = engine.createStatImg();
```

Comment:
Sorry for mentioning, but as I recall it was important to send it with image (:blush:) due to client-server issues (CORS?). But if you it's because of amount of data and/or you know what are you doing here, that's ok.